doc/documentation.md

```diff
@@ -0,0 +1,87 @@
+# PureCloud Streaming Client (SDK)
+
+The PureCloud Streaming Client serves as the Javascript Web SDK. It handles
+establishing and maintaining a websocket connection to the PureCloud streaming
+services. This project uses our new XMPP based signaling for real time APIs.
+
+Features:
+
+- [x] PubSub (notifications)
+- [ ] WebRTC
+    - [ ] Softphone*
+    - [ ] Screen Recording※
+    - [ ] Video/Screen Share※
+    - [ ] ACD Screen Share※
+- [ ] Chat※
+
+\* Coming soon; Until supported in this project, WebRTC softphone applications can be built with
+the [PureCloud WebRTC SDK](https://github.com/mypurecloud/purecloud-webrtc-sdk).
+
+※ Not yet roadmapped
+
+## Usage
+
+The SDK is based on a simple structure of a core client object with a few properties
+and methods, and multiple extensions, each providing their own functionality.
+
+Some extensions are bundled with the client, and others can be added at runtime.
+See [Extensions](extensions.md) for details on implementing or adding an extension (or to
+  request one be added to core).
+
+After creating an instance of the client, your client can add event handlers for
+various events and messages. `connected` and `disconnected`, are examples
+of client events. Extensions are also based on event emitters.
+
+## API
+
+###### Behavior notes
+
+- The client will automatically reconnect when possible. Authentication errors are
+one example of a disconnect that will not trigger a reconnect.
+
+- The client will emit a `connected` event when the connection is established and
+authenticated.
+
+#### Constructor
+
+`new PureCloudStreamingClient(options)`
+
+- parameters
+  - `Object options` with properties:
+    - `String authToken`: Required; access token for the user
+    - `String host`: Required; `wss://streaming.` + `mypurecloud.com || mypurecloud.ie ||
+        mypurecloud.jp || mypurecloud.de || mypurecloud.com.au`
+    - `String jid` : Required; JabberId for the user (get `from api/v2/users/me`)
+
+#### Methods
+
+`client.connect() : Promise<void>` - Initialize the WebSocket connection for streaming
+connectivity with PureCloud. Initialize must be called before any events will trigger.
```

Comment:
I don't see `Initialize` as a method or anything, did you mean `connect`?

Reply:
I think the method _is_ `connect`, the description should probably say 'Initializes the WebSocket...'